src/arr/compiler/anf-loop-compiler.arr

```diff
@@ -101,6 +101,8 @@ j-for = J.j-for
 j-raw-code = J.j-raw-code
 make-label-sequence = J.make-label-sequence
 
+var cases-dispatches = cl-empty
```

Comment:
Would be good to have a comment here explaining the purpose of this variable, and in particular its lifetime: when does it start off as cl-empty, when does it grow, when does it get read, and when does it get reset back to cl-empty?  (I don't see it go back to empty, and that worries me a bit, if we try to compile multiple files in a single run of pyret.jarr: will the cases-dispatches from file1.arr get declared in file2.arr, as a result?